README.md

```diff
@@ -64,86 +97,165 @@ npm install
+@import 'views';
+
+// ------------------------------
+// #SHAME
+// ------------------------------
+@import 'overrides';
+```
+
+See [the UX Pattern Library Files + Application Files guidelines](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#ux-pattern-library-files--application-files) and [example style compile](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#main-style-compile) for more examples/background.
 
 - - -
 
-##Heads Ups
+### Step 4: Configure Settings
+There are a few places you can start to explore configuring aspects of the UX Pattern Library for your purposes. You can find many configurations and base settings in ``src/sass/utilities/_variables.scss`` and can override/customize those in your application's ``utilities.scss`` partial (they'll be used throughout the rest of the compile).
 
-###Pattern Library Elements vs. Reference Site UI
-This repository contains both the code needed to generate pattern library elements as well as a reference site to view/learn about these elements. When and where possible, we've tried to keep the assets needed for both separate by:
+**NOTE:** The code and design contained in the library has not been tested for use alongside other UI Frameworks such as Bootstrap or Foundation.
 
-* prefixing any reference site UI with `pldoc-*` class names in our site templates and Sass/CSS;
-* containing any reference site UI within the ``_src/pl/`` directory.
-* the main pattern library files are contained within the ``_src/pattern-library`` and never compile on their own, but are rather used in this site's and other application's Sass compile files.
+**NOTE:** The source code of the Pattern Library does not include any local or production-focused Sass/Front End tooling. It's expected that your app/site has its own Sass compiler (preferrably LibSass).
 
-- - -
+## Contributions
+Contributions are very welcome. The easiest way is to fork this repo, and then make a pull request from your fork. The first time you make a pull request, you may be asked to sign a Contributor Agreement.
+
+Please refer to our [contributor guidelines](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst) in the main edx-platform repo for important additional information.
+
+### Contributing and the UX Pattern Library
+There are a few additional details alongside our general guidelines to keep in mind contributing to the UX Pattern Library:
+
+#### Pattern Library Features, Ideas, and Improvements
+If you're looking to suggest an idea or you're thinking about developing a feature, start a discussion [by visiting the Open edX JIRA site](https://openedx.atlassian.net/secure/Dashboard.jspa) and  create a new "Issue" by selecting the "Create" button at the top of the page. Choose the project "UX Pattern Library" and the issue type "New Feature" or "Improvement" (you may first need to [create a free JIRA account](https://openedx.atlassian.net/admin/users/sign-up)).
```

Comment:
You should consider linewrapping this file so future changes are easy to see in diff form.